musica/micm/micm.F90

```diff
@@ -63,8 +106,12 @@ subroutine micm_run(time_step, temperature, pressure, dry_air_density, constitue
     real(c_double), dimension(size(constituents, dim=1), &
                               size(constituents, dim=2), &
                               size(constituents, dim=3))   :: c_constituents
+    real(c_double), dimension(size(constituents, dim=1), &
+                              size(constituents, dim=3), &
+                              0)                           :: c_rate_params
```

Comment:
Yes, this should have used `num_layers` as the second dimension (updated). The last dimension will be the number of externally supplied rate constants (e.g., photolysis rate constants), but since we don't have any of these yet, I left the last dimension set to `0`.